core/core.mk

```diff
@@ -83,13 +83,24 @@ libname = utils
 libdir = lib/libutils
 include mk/lib.mk
 
+CFG_CRYPTOLIB_NAME ?= tomcrypt
+CFG_CRYPTOLIB_DIR ?= core/lib/libtomcrypt
+
+CFG_MBEDTLS_MPI ?= y
```

Comment:
I think the symbol name should be more self-explanatory, here it is not clear it is for the TEE core only. `CFG_TEE_CORE_BIGNUM_MBEDTLS[_MPI]" perhaps?
Similarly, I'd rather the subject be "core: switch to mbed TLS for bignum".

Reply:
I was planning letting user TAs switch to use the MPI also using this config variable too. I guess it makes sense to be able to select bignum implementation independently between core and user TAs. In that case I'd prefer using `CFG_CORE_MBEDTLS_MPI`. The rational behind that is that it's something that is enabled in mbedtls when compiled for Core.

We could add another flag for LTC to use MPI instead of MPA or throw all that abbreviations into one big name but that seems a bit clumsy.

This is a transition, once we've stopped using libmpa completely and the dust has settled a bit we'll just remove libmpa and I suppose `CFG_CORE_MBEDTLS_MPI` will look quite natural.

I'll update the subject.

Reply:
> I guess it makes sense to be able to select bignum implementation independently between core and user TAs. 

Indeed it was my main point.

> In that case I'd prefer using `CFG_CORE_MBEDTLS_MPI`.

Fine with me.